app/assets/stylesheets/dm/components/_tags.scss

```diff
@@ -12,7 +12,38 @@
   @include u-padding-y('05');
 }
 
+
+
 // CUSTOM TAGS
+.dm-tag--action-primary {
+  @extend .usa-tag;
```

Comment:
you can remove these changes because we used the existing class

Reply:
Removed.